### DISCUSSION TITLE
Mark the exact object to inspect for Herbiboar

### DISCUSSION THREAD

**Discussion by quintstoffers** [runelite/runelite#15411] [Ideas]
The official mobile client was updated with a Herbiboar helper that marks the exact object to inspect. Could RuneLite get the same capability?

**Comment by Enriath** [comment]
Whether this is possible depends on one of a few things:
* Is there a difference to the client between the correct object and the others?
* Is there a known list of unique tracks that can't be confused? (This would allow us to have unknown for the first couple, and then correct for the rest of the steps)
* If neither of the above, do Jagex happen to transmit whatever they're using for their helper to the Java client?

If any of these work, then sure. Otherwise, unless I missed something (entirely possible), then it wouldn't be feasible to make.
